data/Makefile.am

```diff
@@ -0,0 +1,3 @@
+iconoverridesdir = $(datadir)/eos-application-tools/icon-overrides
+
+iconoverrides_DATA = eos-icon-overrides.ini
```

Comment:
I think this should be `dist_iconoverrides_DATA`, or the file be added to `EXTRA_DIST`